sysboundary/ionosphere.cpp

```diff
@@ -2853,10 +2853,20 @@ namespace SBC {
       }
       Readparameters::get("ionosphere.geometry", this->geometry);
       Readparameters::get("ionosphere.precedence", this->precedence);
+
       uint reapply;
       Readparameters::get("ionosphere.reapplyUponRestart", reapply);
+      this->applyUponRestart = false;
+      if(reapply == 1) {
+         this->applyUponRestart = true;
+      }
+
```

Comment:
Is the indentation here correct? (Or does github's wonderful web interface just fool me?)

Reply:
I'm not seeing anything wrong on github either, what do you mean?